### ISSUE TITLE
Toggle state is not read by screen readers in IE 11.

### ISSUE THREAD

**Issue by sabintim** [OfficeDev/office-ui-fabric-react#10966]
### Environment Information

- **Package version(s)**:  master
- **Browser and OS versions**: IE 11, Windows 10

### Describe the issue:

Toggle state (on, off, changes in state) is not read by screen readers in IE 11.

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fabric#/controls/web/toggle
To repro, open the fabric controls demo page in IE 11 and use Narrator while navigating to the toggle buttons. If you change the role to "checkbox" in DOM, Narrator will announce state 

#### Actual behavior:

Narrator (or NVDA) won't announce the state of the toggle button or the changes in state. 

#### Expected behavior:

Narrator (or NVDA) should announce the state of the toggle button or the changes in state. 

### Documentation describing expected behavior

Toggles are not accessible if the user doesn't know if it's on or off.
https://www.w3.org/TR/wai-aria-1.1/#switch

**Comment by kenotron** [comment]
I've left some comments in the PR before we can accept the change.
